site/content/docs/developing/addons/environment.en.md

```diff
@@ -0,0 +1,273 @@
+# Modeling Additional Environment Resources with AWS CloudFormation
+
+Additional AWS resources, referred to as "addons" in the CLI, are any additional AWS services that a [service or environment manifest](../../manifest/overview.en.md) does not integrate by default. 
+For example, an addon can be a DynamoDB table, an S3 bucket, or an RDS Aurora Serverless cluster that your service needs to read or write to.
+
+You can define additional resources for a workload (such as a [Load Balanced Web Service](../../manifest/lb-web-service.en.md)
+or a [Scheduled Job](../../manifest/scheduled-job.en.md)).
+The lifecycle of workload addons will be managed by the workload and will be deleted once the workload is deleted.
```

Comment:
Should we remove this blurb from the environment addons page?

Reply:
My thought was to include the same high-level description of addons for both workload and environment page, and then say "this page documents workload/env addons". Otherwise, for example if a user lands on the workload addon page, they might think oh addon is for workload-level only without noticing there is another "additional environment resource" page